x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/endpoint_details.tsx

```diff
@@ -167,33 +201,34 @@ export const EndpointDetails = memo(
                   </EuiFlexItem>
                 )}
               </EuiFlexGroup>
-            </>
+            </EuiFlexGroup>
           ),
         },
         {
           title: i18n.translate('xpack.securitySolution.endpoint.details.policyStatus', {
             defaultMessage: 'Policy Response',
           }),
           description: (
-            <EuiHealth
-              color={POLICY_STATUS_TO_HEALTH_COLOR[policyStatus] || 'subdued'}
-              data-test-subj="policyStatusHealth"
+            /** EuiBadge requires additional unnecessary props when the href prop is defined */
```

Comment:
what does it require?  Is it possible to pass empty values so that we can avoid the `@ts-ignore` ?

Reply:
so there's actually a bug where euibadge uses the wrong type when both an onclick and an href are used, it's being tracked in this ticket: https://github.com/elastic/eui/issues/4530

Reply:
thanks!